cypress/support/commands.ts

```diff
@@ -180,8 +276,11 @@ export const getISODateTime = (date: Date) => {
  * @param days number of days to add to the date
  * @returns Date Relative Date Object
  */
-export const addDaysToDate = (date: Date, { days }: { days: number }): Date => {
+export const addSubstractDaysToDate = (date: Date, { days }: { days: number }): Date => {
```

Comment:
```suggestion
export const addSubtractDaysToDate = (date: Date, { days }: { days: number }): Date => {
```

oops, se me fue esto. resulta que esta palabra se escribe ligeramente distinto en inglés. se puede corregir en otro cambio ya para darle merge a este?